roles/installer/templates/deployments/task.yaml.j2

```diff
@@ -322,6 +341,16 @@ spec:
 {% if ee_extra_volume_mounts -%}
             {{ ee_extra_volume_mounts | indent(width=12, first=True) }}
 {% endif %}
+{% if ee_host_network %}
+          hostNetwork: {{ ee_host_network }}
+{% endif %}
+{% if ee_dns_policy %}
+         dnsPolicy: {{ ee_dns_policy }}
+{% endif %}
+{% if ee_dns_policy == 'None' and ee_dns_config is defined }}
```

Comment:
@rakesh561 it looks like there is an extra closing brace here at the end of line 350.